### PR TITLE
Frontend ashraf

### PR DESCRIPTION
**Pull request by Ashraf-Atef1**
adding doctor-user image and more components to src/components.

### CODE REVIEW

File: client/egypt-drug-checker-dashboard/src/components/header/coins/Coins.component.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"use client";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/header/coins/Coins.component.tsx]: use our convention MUtypography

File: client/egypt-drug-checker-dashboard/src/components/header/drop-menu/notifications/Notifications.component.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import Link from "next/link";
+import MUarrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import { Divider as MUdivider, Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/header/drop-menu/notifications/Notifications.component.tsx]: use our convention MUtypography

File: client/egypt-drug-checker-dashboard/src/components/header/drop-menu/profile/Profile.component.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import TextButton from "@/components/shared/text-button/TextButton.component";
+import MUateReviewIcon from "@mui/icons-material/RateReview";
+import MUmanageAccountsIcon from "@mui/icons-material/ManageAccounts";
+import { IconButton, Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/header/drop-menu/profile/Profile.component.tsx]: same comment as before

File: client/egypt-drug-checker-dashboard/src/components/header/profile/Profile.component.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+"use client";
+import UserAvatar from "@/components/shared/user-avatar/UserAvatar.component";
+import { fonts } from "@/utils/theme/fonts";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/header/profile/Profile.component.tsx]: same as before

File: client/egypt-drug-checker-dashboard/src/components/home/content-container/ContentContainer.component.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import { type IcontentContainer } from "./ContentContainer.model";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/home/content-container/ContentContainer.component.tsx]: same as before

File: client/egypt-drug-checker-dashboard/src/components/home/statistics-card/StatisticsCard.component.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
+import { type IstatisticsCard } from "./StatisticsCard.model";
+import CardContainer from "@/components/shared/card-container/CardContainer.component";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/home/statistics-card/StatisticsCard.component.tsx]: same as before

File: client/egypt-drug-checker-dashboard/src/components/home/user/user.component.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import UserAvatar from "@/components/shared/user-avatar/UserAvatar.component";
+import { Iuser } from "./user.model";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/home/user/user.component.tsx]: same as before

File: client/egypt-drug-checker-dashboard/src/components/leader-board/user-card/UserCard.component.tsx [hunk 8]
```diff
@@ -0,0 +1,78 @@
+import CardContainer from "@/components/shared/card-container/CardContainer.component";
+import { IuserCard } from "./UserCard.model";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/leader-board/user-card/UserCard.component.tsx]: same

File: client/egypt-drug-checker-dashboard/src/components/store/store-item/StoreItem.component.tsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
+import CardContainer from "@/components/shared/card-container/CardContainer.component";
+import { type IstoreItem } from "./StoreItem.model";
+import TextButton from "@/components/shared/text-button/TextButton.component";
+import { Typography } from "@mui/material";
```
Comment (KhaledAbdelgalil) [inline client/egypt-drug-checker-dashboard/src/components/store/store-item/StoreItem.component.tsx]: same

**Review by KhaledAbdelgalil — APPROVED** [review]

File: client/egypt-drug-checker-dashboard/src/components/header/coins/Coins.component.tsx (hunk 1 above)
Comment (Ashraf-Atef1) [reply client/egypt-drug-checker-dashboard/src/components/header/coins/Coins.component.tsx]: chore(frontend): adjusting name of the typography component to MUtypography

File: client/egypt-drug-checker-dashboard/src/components/header/drop-menu/profile/Profile.component.tsx (hunk 3 above)
Comment (Ashraf-Atef1) [reply client/egypt-drug-checker-dashboard/src/components/header/drop-menu/profile/Profile.component.tsx]: chore(frontend): adjusting name of the typography and IconButton component to MUtypography and MUiconButton

File: client/egypt-drug-checker-dashboard/src/components/home/content-container/ContentContainer.component.tsx (hunk 5 above)
Comment (Ashraf-Atef1) [reply client/egypt-drug-checker-dashboard/src/components/home/content-container/ContentContainer.component.tsx]: chore(frontend): adjusting name of the typography component to MUtypography

File: client/egypt-drug-checker-dashboard/src/components/leader-board/user-card/UserCard.component.tsx (hunk 8 above)
Comment (Ashraf-Atef1) [reply client/egypt-drug-checker-dashboard/src/components/leader-board/user-card/UserCard.component.tsx]: chore(frontend): adjusting name of the typography component to MUtypography

**Pull request merged**